Source/WebCore/platform/cocoa/PlaybackSessionModelMediaElement.mm

```diff
@@ -87,7 +87,13 @@
             audioTracks->removeEventListener(events.removetrackEvent, *this, false);
         }
 
-        if (auto* textTracks = oldMediaElement->audioTracks()) {
+        if (auto* videoTracks = oldMediaElement->videoTracks()) {
```

Comment:
Isn’t this part of the other patch?

Reply:
it is, I've added the two patches to build in the PR. Need to limit to the top commit in the stack. I'll repush once the other land .